spec/index.bs

```diff
@@ -632,6 +653,26 @@ dictionary GPUBufferDescriptor : GPUObjectDescriptorBase {
         </div>
 </dl>
 
+## Buffer Destruction ## {#buffer-destruction}
+
+An application that doesn't need a {{GPUBuffer}} can choose to lose access to
+it before garbage collection by calling {{GPUBuffer/destroy()}}. Non normatively
+this allows the user-agent to reclaim the memory associated with the {{GPUBuffer}}
```

Comment:
let's clarify that we are talking about GPU memory here (even though it could be the same memory)